examples/vsphere/sample-terraform-dhcp.tfvars

```diff
@@ -99,21 +99,20 @@ node_pools = {
   system = {
     count       = 1
     cpus        = 8
-    memory      = 16384
+    memory      = 65536
```

Comment:
Wow! This is 4x the original value. Is this for logging/monitoring/etc? Did not realize the work load we've been pushing to the customer had grown that large.

Reply:
That's going the off the sizing teams recommendation for a "Small Sized" deployment based of their performance tests. Not entirely sure what specifically will be consuming that RAM, but I also see that's recommendation across all the clouds.